modules/kubernetes/grafana-agent/main.tf

```diff
@@ -129,3 +134,15 @@ resource "helm_release" "grafana_agent_extras" {
   max_history = 3
   values      = [local.extras_values]
 }
+
+
+resource "helm_release" "kube_state_metrics" {
```

Comment:
Are we sure that our tenants always wants this feature?
I would suggest to add it as a opt in feature.